.github/workflows/build-linux.yml

```diff
@@ -8,16 +8,21 @@ on:
   pull_request:
     branches:
       - master
+      - development
 
 jobs:
   linux:
-    name: "Linux: PHP v${{ matrix.php }}"
-    runs-on: ubuntu-latest
+    name: "${{ matrix.os }}: PHP v${{ matrix.php }}"
+    runs-on: ${{ matrix.os }}
 
     strategy:
       fail-fast: false
 
       matrix:
+        os:
+          - ubuntu-18.04
+          - macOS-latest
```

Comment:
OS can be lowercase I think.

Reply:
See release notes: https://github.com/shivammathur/setup-php/releases

Reply:
@ruudboon Somehow it works. Did you see? :)

Reply:
Haha yes. For now